src/Microsoft.ML.StandardTrainers/StandardTrainersCatalog.cs

```diff
@@ -724,19 +724,20 @@ private static ICalibratorTrainer GetCalibratorTrainerOrThrow(IExceptionContext
         }
 
         /// <summary>
-        /// Predicts a target using a linear multiclass classification model trained with the <see cref="OneVersusAllTrainer"/>.
+        /// Create <see cref="OneVersusAllTrainer"/>, which predicts a multiclass target using one-versus-all strategy with
```

Comment:
>  [](start = 18, length = 1)

"a"

Reply:
added

---
In reply to: [277761844](https://github.com/dotnet/machinelearning/pull/3516#discussion_r277761844) [](ancestors = 277761844)